### DISCUSSION TITLE
new migrations not migrating in heroku

### DISCUSSION THREAD

**Discussion by bytesagar** [prisma/prisma#12167] [Q&A]
I deleted my migrations folder and created new migrations but when I try to push it in heroku it is showing error  that the model was already created something like that. how do I solve this.

**Comment by nurul3101** [comment]
Hey @bytesagar 👋 
What is the exact error message that you are getting?
When you deleted your migrations folder were you trying to reset your database?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
